interfaces/IDRM.h

```diff
@@ -47,13 +47,7 @@
 #include <typeinfo>
 #include <vector>
 
-#ifdef __GNUC__
```

Comment:
Hopefully we can do without this change. This change makes an arbitrary interface secification dependend on Thunder. We should aim to make sure that these interfaces do not pull in any Thunder related components if possible. So this "Copy-Paste" of these preprocessor statements is a bit deliberat to make sure the file is agnostic to Thunder.

Reply:
as we discussed moved this to Portability.h